### DISCUSSION TITLE
screen shaking vertically

### DISCUSSION THREAD

**Discussion by chiragwad** [Eloston/ungoogled-chromium#1328] [Q&A]
i noticed that my screen is shaking while i am browsing reddit, i thought i was imagining it but no it was real. i am using UC Version 87.0.4280.67. anyone knows why this might be happening and how can i fix it?

**Comment by chiragwad** [comment]
i found that it was a bug in very old builds in chrome. i guess i have to live with it

**Comment by Eloston** [comment]
Have you enabled any of the fingerprint deception flags in `chrome://flags`? They are known to cause visual glitches sometimes.

**Comment by chiragwad** [comment]
yes i have. i will try disabling them

**Comment by chiragwad** [comment]
@Eloston do you know if the chocolatey install of UC is safe.?i am not very knowledgeable and cant compile the application

**Reply by SwiftCoderJoe** [reply]
Yes, it should be safe.
